RHIDP-12605: Phase 2 Homepage - persona support and related RBAC (#2153)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/configure_customizing-rhdh/assembly-deploy-persona-specific-homepages-to-provide-targeted-content-to-distinct-teams.adoc

:_mod-docs-content-type: ASSEMBLY
ifdef::context[:parent-context: {context}]

[id="deploy-persona-specific-homepages-to-provide-targeted-content-to-distinct-teams_{context}"]
= Deploy persona-specific homepages to provide targeted content to distinct teams

:previouscontext: {context}
:context: deploy-persona-specific-homepages-to-provide-targeted-content-to-distinct-teams

[role="_abstract"]

> Deploy persona-specific homepage layouts in {product} to deliver targeted content to distinct user groups by configuring visibility rules that automatically show role-appropriate cards, templates, and quick access links to each team.

Reviewer: ```suggestion
Persona-specific homepage layouts allow you to deliver targeted content, such as role-appropriate templates and links, to distinct user groups in {product}.
```

> Deploy persona-specific homepage layouts in {product} to deliver targeted content to distinct user groups by configuring RBAC group-based visibility rules that automatically show role-appropriate cards, templates, and quick access links to each team.

Reviewer 2: It's technically a Group of the user, not an RBAC group. RBAC group could result in confusions with RBAC Policies, or? Wdty @jmagak @karthikjeeyar 

```suggestion
Deploy persona-specific homepage layouts in {product} to deliver targeted content to distinct user groups by configuring visibility rules that automatically show role-appropriate cards, templates, and quick access links to each team.
```
